easypost/util.py

```diff
@@ -45,7 +43,7 @@ def get_lowest_object_rate(
     return lowest_rate
 
 
-def get_lowest_smart_rate(smart_rates, delivery_days: int, delivery_accuracy: str) -> Rate:
```

Comment:
# Reviewer Note

These type hints were removed due to circular import issues. Will try and revisit this when I come back and fix all type hints

Reply:
Put single quotes around the word `Rate`

Reply:
It's the import itself that is a problem. I already tried that approach